backend/src/main/java/com/redhat/cloud/notifications/routers/internal/errata/ErrataMigrationRepository.java

```diff
@@ -0,0 +1,100 @@
+package com.redhat.cloud.notifications.routers.internal.errata;
```

Comment:
Even if it's temporary, this bean should be part of `com.redhat.cloud.notifications.db.repositories` package.

Reply:
Moved!